include/itkCoocurrenceTextureFeaturesImageFilter.h

```diff
@@ -103,13 +103,16 @@ class ITK_TEMPLATE_EXPORT CoocurrenceTextureFeaturesImageFilter
   typedef TInputImage                     InputImageType;
   typedef TOutputImage                    OutputImageType;
   typedef TMaskImage                      MaskImageType;
-  typedef TInputImage                     DigitizedImageType;
 
   typedef typename InputImageType::PixelType    PixelType;
   typedef typename MaskImageType::PixelType     MaskPixelType;
   typedef typename InputImageType::IndexType    IndexType;
   typedef typename InputImageType::PointType    PointType;
 
+
+  typedef int                                                            HistogramIndexType;
+  typedef itk::Image< HistogramIndexType, TInputImage::ImageDimension >  DigitizedImageType;
```

Comment:
It is preferred to make these typedefs `private` or `protected` so that the value can be changed in the future without worrying about breaking API backwards compatibility.

Reply:
The problem is that it's used NeighborhoodIteratorType that need to be public to allow to access the NeighborhoodRadius variable. Can it still be either protected or private?